DALK.PL_ANALYZER/Models/Matches/MatchModelView.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DALK.PL_ANALYZER.Models.Matches
+{
+    public class MatchModelView
+    {
+        public bool IsPlayedMatch { get; private set; }
+        public string League { get; private set; }
+        public string Season { get; private set; }
+        public string DateTime { get; private set; }
+        public string HomeIconUrl { get; private set; }
+        public string Home { get; private set; }
+        public string AwayIconUrl { get; private set; }
+        public string Away { get; private set; }
+        public string MatchDescription { get; private set; }      
+        public string GameTitle { get; private set; }
+        public int HomePoints { get; set; }
+        public int AwayPoints { get; set; }
+        public string WinnerIconUrl { get; private set; }
+        public int WinnerPoints { get; private set; }
+        public string Winner { get; private set; }
+        public string LooserIconUrl { get; private set; }
+        public int LooserPoints { get; private set; }
+        public string Looser { get; private set; }
+        public string MVP { get; set; }
+
+        public MatchModelView(Match m)
+        {
+            IsPlayedMatch = m is PlayedMatch;
+            League = m.League.ToString();
+            Season = m.Season.ToString();
+            DateTime = m.DateTime.ToString("yyyy-MM-dd HH:mm:ss");
+            HomeIconUrl = m.Home.IconUrl;
+            AwayIconUrl = m.Away.IconUrl;
+            Home = m.Home.ToString();
+            Away = m.Away.ToString();
+            MatchDescription = m.MatchDescription;
+
+            if (m.Stage is GroupStage)
```

Comment:
Use polymophism to return proper GameTitle from stage children classes and dont do it here. 
Notice that all the data you get are from the stage classes, so let them do the actual job/logic